app/modules/user/stats.rb

```diff
@@ -284,6 +284,11 @@ def returned_affiliate_fee_cents_due_to_refunds_and_chargebacks(balance_ids)
     refunded_affiliate_fee_cents + disputed_affiliate_fee_cents
   end
 
+  def carried_over_cents_from_failed_payout_for_balances(balance_ids)
+    failed_balance_ids = payments.failed.joins(:balances).where(balances: { id: balance_ids }).pluck("balances.id").uniq
+    Balance.where(id: failed_balance_ids).sum(:amount_cents)
```

Comment:
We also need to track which payment the carried-over balance is from (failed payout date shown in the design), also handle case where there were multiple failed attempts for the same balance etc.